style.css

```diff
@@ -14,34 +14,28 @@ html {
 }
 
 body {
-   padding: 0 4rem;
    font-size: 2rem;
-   background-color: #F5FFFA;
+   background-color: #fff;
    font-family: 'Open Sans', sans-serif;
-   /*-webkit-column-count: 2;
-   -webkit-column-gap: 4rem;
-   -webkit-column-rule-style: solid;
-   -webkit-column-rule-width: 1px;
-   -webkit-column-rule-color: #000;
-   -webkit-column-rule: 1px solid #000;*/
 }
 
-section #header {
-   background-color: #fff;
-   margin-bottom: 4rem 0;
+header .site-header {
+   background-color: #000;
+   opacity: .5px;
+   position: fixed;
 }
 
-h1 {
-   font-family: 'Over the Rainbow';
-   color: #1E90FF;
-   float: left;
+.site-header h1 {
+   float: left !important;
```

Comment:
Don't use important unless you really really really have to. The fact that you need to here means there's something off. You should look into getting those out.